components/Chips/examples/ChipsFilterExampleViewController.m

```diff
@@ -34,6 +35,13 @@ - (id)init {
   return self;
 }
 
+- (MDCContainerScheme *)containerScheme {
+  MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+  scheme.colorScheme = self.colorScheme;
```

Comment:
Would you please add the typography scheme either here or as a follow?

Reply:
done